graphalyzer-server/src/file_handler/os/OSWrapper.py

```diff
@@ -49,7 +49,7 @@ def getOSPathHeader(self):
 			return "c:/"
 		else:
 			self.logger.debug("OS is Linux. Returning path header '/'")
-			return "/"
```

Comment:
It's quite minor but it's no longer returning the path header that it says it is in the logger message. Instead it's returning an empty string.